test/src/test/java/org/corfudb/integration/CorfuReplicationClusterConfigIT.java

```diff
@@ -2294,9 +2294,8 @@ public void testSinkLockRelease() throws Exception {
         clearLockTable(false);
         log.info("Sink's lock table cleared!");
 
-        // Wait till the lock release is asynchronously processed and the replication status on Source changes to
-        // STOPPED
-        countDownLatch.await();
+        // Allow for the sinks leadership loss to be processed
+        Thread.sleep(PARAMETERS.TIMEOUT_NORMAL.toMillis());
```

Comment:
Just curious....Any reason for this change?

Reply:
Discussed offline